### PR TITLE
chore: add `SetContains` API

### PR DESCRIPTION
**Pull request by poppoerika**
Added `SetContainsElement` and `SetContainsElements` API reference to the doc.

Ticket: https://github.com/momentohq/dev-eco-issue-tracker/issues/208

### CODE REVIEW

File: docs/develop/api-reference/set-collections.md [hunk 1]
```diff
@@ -117,4 +117,49 @@ Removes multiple elements from an existing set item.
...
\ No newline at end of file
+</details>
+
+### SetContainsElement
+Checks if a provided element is in the given set.
+
+| Name      | Type            | Description                                   |
+|-----------| --------------- | --------------------------------------------- |
+| cacheName | String          | Name of the cache.                            |
+| setName   | String          | Name of the set item to be altered.           |
```
Comment (allenheltondev) [inline docs/develop/api-reference/set-collections.md:128]: Since this operation does not modify the set, we should change the description to `Name of the set item`

File: docs/develop/api-reference/set-collections.md [hunk 2]
```diff
@@ -117,4 +117,49 @@ Removes multiple elements from an existing set item.
...
+| element   | String \| bytes | Element to be checked its existence by this operation.   |
```
Comment (allenheltondev) [inline docs/develop/api-reference/set-collections.md:129]: I'd rephrase this description as `Name of the element to check existence of`

File: docs/develop/api-reference/set-collections.md [hunk 3]
```diff
@@ -117,4 +117,49 @@ Removes multiple elements from an existing set item.
...
+
+<details>
+  <summary>Method response object</summary>
+
+The response object for SetContainsElement returns three possible options, a cache hit, miss, or an error.
+
+* Hit
```
Comment (allenheltondev) [inline docs/develop/api-reference/set-collections.md:136]: Below you have a nested bullet point with `containsElements(): bool[]`

I believe the Hit for this operation has a `containsElement: bool` response

File: docs/develop/api-reference/set-collections.md [hunk 4]
```diff
@@ -117,4 +117,49 @@ Removes multiple elements from an existing set item.
...
+* Miss
+* Error
+
+See [response objects](./response-objects.md) for specific information.
+
+</details>
+
+### SetContainsElements
+Checks if provided elements is in the given set.
```
Comment (allenheltondev) [inline docs/develop/api-reference/set-collections.md:145]: Since *elements* is plural, we need to update the verbiage to"

```
Checks if provided elements are in the given set.
```

File: docs/develop/api-reference/set-collections.md [hunk 5]
```diff
@@ -117,4 +117,49 @@ Removes multiple elements from an existing set item.
...
+
+| Name      | Type       | Description                         |
+|-----------|------------|-------------------------------------|
+| cacheName | String     | Name of the cache.                  |
+| setName   | String     | Name of the set item to be altered. |
```
Comment (allenheltondev) [inline docs/develop/api-reference/set-collections.md:150]: I'd remove "to be altered" from this description

File: docs/develop/api-reference/set-collections.md [hunk 6]
```diff
@@ -117,4 +117,49 @@ Removes multiple elements from an existing set item.
...
+| elements  | String[] \ | bytes[]                             | Elemenst to be checked its existence by this operation.   |
```
Comment (allenheltondev) [inline docs/develop/api-reference/set-collections.md:151]: I'd update the description here to:

```
Array of element names to check existence of.
```

**Review by allenheltondev — CHANGES_REQUESTED** [review]

**Comment by poppoerika** [comment]
Closing this PR since this is included in this PR: https://github.com/momentohq/public-dev-docs/pull/131

**Comment by poppoerika** [comment]
Closing this PR since this is included in this PR: https://github.com/momentohq/public-dev-docs/pull/131
